code/game/turfs/closed/minerals.dm

```diff
@@ -29,9 +29,6 @@
 
 /turf/closed/mineral/Initialize(mapload)
 	. = ..()
-	var/matrix/M = new
-	M.Translate(-4, -4)
-	transform = M
```

Comment:
Could you add a comment about this? We may or may not need to do something about it in future, and I don't want to forget.
somethin like
// Wallening todo: figure out if lavaland walls need offsets or not
would do

Reply:
ah hell